lib/AFS/CellCC/DB.pm

```diff
@@ -484,6 +485,64 @@ update_job(%) {
     _job_set(%info);
 }
 
+# Similar to update_job but updates several jobs at once.
+sub
+update_jobs($%) {
```

Comment:
I had previously mentioned that we should maybe have a db primitive to update several jobs at once, to avoid those loops in all of the callers. But I meant having another SQL query to do all of the updates at once, not just having a function that calls _job_set in a loop. I don't think this function is worth it; it adds some new code but it's almost the same as just update_job(); I'd rather have the loops in the caller.

Actually having some SQL query do the updates all at once is maybe desirable at some point, but after thinking about it, I don't think it's worth the complexity right now; speed here isn't important right now.